build/esbuild-build.ts

```diff
@@ -17,10 +17,26 @@ export const esBuildContext: esbuild.BuildOptions = {
     ".ttf": "dataurl",
     ".svg": "dataurl",
   },
-  outdir: "static/out",
+  outfile: "static/bundles/bundles.js",
+  entryNames: "bundles", // Ensure the CSS is named bundles.css
   define: createEnvDefines(["SUPABASE_URL", "SUPABASE_ANON_KEY"], {
     commitHash: execSync(`git rev-parse --short HEAD`).toString().trim(),
   }),
+  plugins: [
+    {
+      name: "css-bundle",
+      setup(build) {
+        build.onEnd((result) => {
+          execSync(`cat static/styles/rewards/pay.css > static/bundles/bundles.css`);
+          execSync(`cat static/styles/rewards/background.css >> static/bundles/bundles.css`);
+          execSync(`cat static/styles/toast.css >> static/bundles/bundles.css`);
+          execSync(`cat static/styles/rewards/claim-table.css >> static/bundles/bundles.css`);
+          execSync(`cat static/styles/rewards/media-queries.css >> static/bundles/bundles.css`);
+          execSync(`cat static/styles/rewards/light-mode.css >> static/bundles/bundles.css`);
```

Comment:
This seems wrong. This should be done dynamically

Reply:
@0x4007 This way I wanted to generate the exact look as the original code because there are some conflicting styles within the files and If two CSS styles conflict with each other, the one that is applied last in the HTML (or the last one loaded in the stylesheet) will generally take precedence due to the cascading nature of CSS.
I can investigate this further to find a better ways or I can revert this to the previous version if it was good enough.

Reply:
That is a good point to specify the order. Ideally if we resolved the conflicts (perhaps there is some automated tooling, like Chrome's Lighthouse that can solve for this?)

Reply:
I had the same problem when rebuilding for [`NextJS`](https://github.com/ubiquity/safe.ubq.fi/blob/c2eb9823ed6836694ca4150df5101d3eeaba9372/app/globals.css). My implementation caused a few changes too though so it's not a good point of ref for what to do but I'd appreciate a working solution as it would save me time down the line

Reply:
> Does this clear the file contents on every build? It should so that the file length doesn't run forever.   


As you can observe, the first line clears the file (the > symbol indicates creation), while the following lines add content to it (using the >> symbol for appending):
```         execSync(`cat static/styles/rewards/pay.css > static/bundles/bundles.css`);```